index.html

```diff
@@ -97,86 +96,156 @@ <h1>bam2.iobio</h1>
         <div class="percent-row">
 
           <iobio-panel>
-            <iobio-percent-box
-              label="Mapped Reads"
-              percent-key="mapped_reads"
-              total-key="total_reads">
-            </iobio-percent-box>
+            <iobio-info-button label="Mapped Reads">
```

Comment:
We should try to remove the duplication between the label and the header slot. We should only need one or the other I think